source/includes/stackpath/_dns_zones.md

```diff
@@ -107,6 +107,68 @@ Attributes | &nbsp;
 `verified`<br/>*string* | The date a zone's nameservers were last audited by StackPath.
 `status`<br/>*string* | The status of the zone. It can either be `ACTIVE`, `INACTIVE`.
 
+
+
+<!-------------------- CREATE A DNS ZONE -------------------->
+
+### Create a DNS zone
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/dnszones"
+```
+> Request body example for without DNS records to import:
+
+```json
+{
+  "domain":"my-domain.com"
+}
+```
+> Request body example for with DNS records to import:
+
+```json
+{
+  "domain": "my-domain.com",
+  "records": [
+    {
+      "name": "*",
+      "type": "NS",
+      "ttl": 3600,
+      "data": "ns1.above.com.",
+      "weight": 1
+    }
+  ]
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnszones</code>
+
+Create a DNS zone in a given [environment](#administration-environments).
+
+Required | &nbsp;
+------- | -----------
+`domain`<br/>*string* | The name of the zone.
```

Comment:
String instead of string

Reply:
6afe84ddad92fdfd93d1689a64645bf5c4d35c3a